### PR TITLE
Adding support for single axis DPAD's

### PR DESCRIPTION
**Pull request by arzulo**
Adds in a new virtual button typing created through `.dpad_1axis`. 
This creates 2-buton DPAD's in either the vertical or horizontal direction.

I can follow up with documentation once changes are approved. 

See for conversation in discord: https://discord.com/channels/724320164371497020/725060331759730738/1182881561566253086

### CODE REVIEW

**Comment by arzulo** [comment]
2nd commit ditches the single `.dpad_1axis` method previously mentioned and splits the creation of the horizontal / vertical type DPADs into 2 seperate methods.  `.hpad` and `.vpad`.  Helps with clarity

File: scripts/__input_class_virtual/__input_class_virtual.gml
Comment (offalynne) [inline scripts/__input_class_virtual/__input_class_virtual.gml:1]: The switch format is a bit verbose here. Can you drop the `direction` var and rewrite as if/else?

**Comment by arzulo** [comment]
Sorry @offalynne ! Didn't think to ask about using 6.2.  
Merged 6.2 into my work and handled conflicts. Added a little more debug information in the vbutton tester object too.

File: scripts/__input_class_virtual/__input_class_virtual.gml
Comment (arzulo) [reply scripts/__input_class_virtual/__input_class_virtual.gml:1]: Done

**Pull request merged**